regression/cbmc/array_constraints1/test.desc

```diff
@@ -4,6 +4,6 @@ main.c
 ^EXIT=10$
 ^SIGNAL=0$
 ^VERIFICATION FAILED$
-^\*\* 2 of 14
+^\*\* 2 of \d+
```

Comment:
Why `\d+` ? shouldn't the number of VCs be known and static?

Reply:
Agree with Martin here - it's probably good that the number of VC's should be checked in the tests. I appreciate that is an added maintenance issue but we wouldn't want to silently regress the number of VCs either